### PR TITLE
chore(issues): Remove usage of issue-alert-fallback-targeting flag from the frontend

### PR DESCRIPTION
**Pull request by leeandher**
It's GA'd so I'll delete it from the frontend and try to get rid of any unused components as well.

### CODE REVIEW

**Review by vartec — APPROVED** [review]

File: static/app/views/alerts/rules/issue/ruleNode.tsx [hunk 1]
```diff
@@ -494,8 +493,7 @@ function RuleNode({
 
     if (
       data.id === IssueAlertActionType.NOTIFY_EMAIL &&
-      data.targetType === MailActionTargetType.ISSUE_OWNERS &&
-      !organization.features.includes('issue-alert-fallback-targeting')
```
Comment (scttcper) [inline static/app/views/alerts/rules/issue/ruleNode.tsx:498]: i think this block should be deleted?

File: static/app/views/projectInstall/issueAlertOptions.tsx [hunk 2]
```diff
@@ -237,9 +237,7 @@ class IssueAlertOptions extends DeprecatedAsyncComponent<Props, State> {
       actions: [
         {
           ...ISSUE_ALERT_DEFAULT_ACTION,
-          ...(this.props.organization.features.includes('issue-alert-fallback-targeting')
-            ? {fallthroughType: 'ActiveMembers'}
-            : {}),
+          ...{fallthroughType: 'ActiveMembers'},
```
Comment (scttcper) [inline static/app/views/projectInstall/issueAlertOptions.tsx:240]: i think in this case we can change ISSUE_ALERT_DEFAULT_ACTION to have `fallthroughType: 'ActiveMembers'` all the time

File: static/app/views/alerts/rules/issue/ruleNode.tsx (hunk 1 above)
Comment (scttcper) [reply static/app/views/alerts/rules/issue/ruleNode.tsx:498]: yeah we do not want to display the below MarginlessAlert

**Review by scttcper — CHANGES_REQUESTED** [review]
just blocking to make sure we don't auto merge or whatever

**Review by scttcper — APPROVED** [review]
nice, one less api call this is great

File: static/app/views/settings/project/projectOwnership/index.spec.tsx [hunk 3]
```diff
@@ -140,16 +140,12 @@ describe('Project Ownership', () => {
       });
     });
 
-    it('should hide issue owners for issue-alert-fallback-targeting flag', () => {
-      const org = {
-        ...organization,
-        features: ['issue-alert-fallback-targeting'],
-      };
+    it('should hide issue owners', () => {
```
Comment (scttcper) [inline static/app/views/settings/project/projectOwnership/index.spec.tsx:143]: just realizing this test is probably useless

**Pull request merged**